qiskit/_jobprocessor.py

```diff
@@ -75,7 +75,9 @@ def run_remote_backend(qobj, api, wait=5, timeout=60):
     if 'error' in output:
         raise ResultError(output['error'])
 
+    logger.info('Running on remote backend ' + qobj['config']['backend'] + ' with job id: ' + output['id'])
```

Comment:
Would it be possible to drop the level to `logger.debug()`, in the hopes of preserving the original output while we are still deciding the right amount of verbosity?

Also, could we try to make use of the logging default parameter passing and change it to something similar to:
```python
logger.debug('Running on remote backend %s with job id: %s',
             qobj['config']['backend'], output['id'])
```

(and thanks for the quick adjustment to using logging instead of print!)

Reply:
Yes sure :).